server/controllers/product.js

```diff
@@ -0,0 +1,47 @@
+const axios = require('axios');
+const path = require('path');
+require('dotenv').config({ path: path.resolve(__dirname, '../../.env') });
+
+exports.products = (req, res) => {
+
+  var id = req.params.id;
+
+  var productConfig = {
+    method: 'get',
+    url: `/products/${id}`,
+    baseURL: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp',
+    headers: {'Authorization': process.env.API_KEY}
+  };
+
+  axios(productConfig)
+  .then((response) => {
+    // console.log(response.data, '===========getProduct response data')
+    res.json(response.data);
+  })
+  .catch((error) => {
+    console.log(error);
```

Comment:
make sure you handle the error too rather than just logging it, otherwise looks good so far

Reply:
Thanks! I'll fix it in next PR.